tests/31sync/15lazy-members.pl

```diff
@@ -99,9 +99,21 @@
             )
          }, foreach => [ 1 .. 10 ])
       })->then( sub {
+         # Ensure synapse has propagated this message to Alice's synchotron
```

Comment:
```suggestion
         # Ensure synapse has propagated this message to Alice's sync stream
```

"synchrotron" is a bit of an unofficial name that we mostly just use to describe sync workers on Synapse.